### PR TITLE
✨ spotlight backend sorting

### LINKED ISSUE #1848: Spotlight sorting made through backend - Rubick
Following here https://github.com/kodadot/nft-gallery/pull/1815 would be nice to have sorting done on Rubick, which mean not sorting on the client but on the backend 

- https://github.com/kodadot/nft-gallery/pull/1815
- https://github.com/kodadot/rubick/pull/18

### PR DESCRIPTION
**Pull request by roiLeo**
I'm not able to sort on averagePrice column yet.

### PR type

- [x] Bugfix
- [x] Feature

### What's new?

- [x] PR closes #1848 
- [x] backend sorting on Spotlight page

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality

### CODE REVIEW

**Comment by yangwao** [comment]
Is it related? https://github.com/kodadot/rubick/pull/23

File: components/spotlight/SpotlightDetail.vue [hunk 1]
```diff
@@ -24,14 +25,15 @@ type NftSimpleView = {
...
-export default class SpotlightDetail extends Vue {
+export default class SpotlightDetail extends mixins(PrefixMixin) {
   @Prop(String) public account!: string
   protected nfts: NftSimpleView[] = []
   protected isLoading = true
 
   protected async fetchNFT(account: string) {
     const nfts = await this.$apollo.query({
       query: nftSimpleListByAccount,
+      client: this.urlPrefix,
```
Comment (vikiival) [inline components/spotlight/SpotlightDetail.vue:36]: spotlight has nothing to do with prefixmixin

**Review by vikiival — CHANGES_REQUESTED** [review]
👀

File: components/spotlight/SpotlightDetail.vue (hunk 1 above)
Comment (roiLeo) [reply components/spotlight/SpotlightDetail.vue:36]: @vikiival it throw this without client
Capture d’écran 2022-01-19 à 8 21 58 AM

**Comment by roiLeo** [comment]
> Is it related? [kodadot/rubick#23](https://github.com/kodadot/rubick/pull/23)

This is related to #1893 but yes we can try the same with spotlight resolver.

File: components/spotlight/SpotlightDetail.vue (hunk 1 above)
Comment (vikiival) [reply components/spotlight/SpotlightDetail.vue:36]:
```suggestion
      client: 'subsquid',
```

Comment (roiLeo) [reply components/spotlight/SpotlightDetail.vue:36]: not subsquid, rmrk

Comment (roiLeo) [reply components/spotlight/SpotlightDetail.vue:36]: pushed some changes

**Comment by roiLeo** [comment]
shiiiiit

**Pull request merged**